docs/guide/tutorial-console.md

```diff
@@ -107,6 +107,21 @@ You can see an example of this in the advanced project template.
 > ```
 
 
+Bash completion
+---------------
+
+Auto-completion of command arguments is a useful thing when working with the shell. Since version 2.0.11, the `./yii` command provides auto completion
+for the bash out of the box. This is provided by the `yii.bash-completion` file located in the framework base directory.
+For temporary usage you can source that file by typing:
+
+    . yii.bash-completion
```

Comment:
I'd prefer source yii.bash-completion instead of . yii.bash-completion as it is more readable and less confusing.